src/signals/incident/containers/IncidentContainer/index.test.js

```diff
@@ -5,24 +5,38 @@ import { withAppContext } from 'test/utils';
 import { IncidentContainer, mapDispatchToProps } from './index';
 import { GET_CLASSIFICATION, UPDATE_INCIDENT, CREATE_INCIDENT } from './constants';
 
-// jest.mock('../../components/IncidenptWizard', () => () => 'IncidentWizard');
 jest.mock('signals/incident/components/form/MapInput', () => () => 'MapInput');
 
 describe('<IncidentContainer />', () => {
   let props;
+  let origSessionStorage;
 
   beforeEach(() => {
     props = {
       incidentContainer: { incident: {} },
       getClassification: jest.fn(),
       updateIncident: jest.fn(),
       createIncident: jest.fn(),
-      isAuthenticated: false
+    };
+    origSessionStorage = global.sessionStorage;
+
+    global.sessionStorage = {
```

Comment:
why do you not use `getAccessToken.mockImplementation(() => 'mock')` here to mock api service?

Reply:
See previous comment